test/jdk/java/lang/StringBuilder/StressSBTest.java

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8351443
+ * @summary Randomized test of StringBuilder
+ * @run main StressSBTest
+ */
+
+import java.lang.System.Logger;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.nio.CharBuffer;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+
+public class StressSBTest {
+
+    // Logger for info and errors
+    static final Logger LOG = System.getLogger("StressSBTest");
+    // Number of concurrent platform threads
+    private final static int N_THREADS = 11;
+    // Base length of test run
+    private final static Duration N_Duration = Duration.ofSeconds(2);
+    // Cache jtreg timeout factor to allow test to be run as a standalone main()
+    private static final double TIMEOUT_FACTOR =
+            Double.parseDouble(System.getProperty("test.timeout.factor", "1.0"));
+
+    /**
+     * Run the stress test with a fixed set of parameters.
+     *
+     * @param ignored
+     */
+    public static void main(String[] ignored) {
+        Duration duration = Duration.ofMillis((long) (N_Duration.toMillis() * TIMEOUT_FACTOR));
+        new StressSBTest().stress(N_THREADS, duration);
+    }
+
+    /**
+     * Stress test using a number of platform threads for a duration.
+     * Success is marked by a lack of exceptions.
```

Comment:
If I'm reading the test correctly, then the exceptions that happen in the launched thread are merely logged and thus the test will never throw any exceptions and will never fail?